types/recharts/index.d.ts

```diff
@@ -271,14 +272,14 @@ export interface LineProps {
 	dot?: boolean | any | React.ReactElement<any> | RechartsFunction;
 	activeDot?: boolean | any | React.ReactElement<any> | RechartsFunction;
 	label?: boolean | any | React.ReactElement<any> | RechartsFunction;
-	points: any[];
+	points?: any[];
 	layout?: LayoutType;
 	connectNulls?: boolean;
 	unit?: string | number;
 	name?: string | number;
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?